src/api/swagger-spec/swagger.yml

```diff
@@ -38,47 +38,94 @@ paths:
             type: object
             properties:
               data:
-                $ref: "#/definitions/chart"
+                $ref: "#/definitions/resource"
         default:
           description: unexpected error
           schema:
             $ref: "#/definitions/error"
 definitions:
-  chartAttributes:
+  resource:
+    type: object
+    required:
+      - type
+      - id
+      - attributes
+    properties:
+      type:
+        type: string
+        minLength: 1
+      id:
+        type: string
+        minLength: 1
+      links:
+        type: object
+      attributes:
+        type: object
+  chartVersion:
```

Comment:
I think at this time where we put the readme.md is implementation-TBD. It could be a part of the above model, or it could be put as an `attribute` key/val into `resource` model as we prepare the HTTP response.